includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -585,6 +585,18 @@ public function has_allowed_items_in_cart() {
 				return false;
 			}
 
+			/**
+			 * Filter whether product supports Payment Request Button on cart page.
+			 *
+			 * @since x.x.x
```

Comment:
this needs to be updated.

Reply:
+1

Reply:
@dpaun1985 What version should I change this to? I've checked the allow maintainers to edit button on the PR to allow folks to modify this once the PR is added to a milestone.

Reply:
ok great. let's leave it like this for now. LGTM.